### DISCUSSION TITLE
Unused 11ty dependencies

### DISCUSSION THREAD

**Discussion by genemars** [11ty/eleventy#2239] [Q&A]
Why is `vue` plugin built as a dependency in the 11ty core?
Is Eleventy going to be a vue-based product?
Should it just be an external plugin?

**Comment by genemars** [comment]
Searching in the code I see it's only used in the tests for a plugin that it's not there.
So I am wondering if these `devDependencies` can be removed and tests moved to the eleventy-vue plugin repository?

**Reply by zachleat** [reply]
Hm? Are they installing when you npm install?

**Reply by genemars** [reply]
Yes, working with this repository, they are installed.
![Schermata da 2022-02-22 01-18-12](https://user-images.githubusercontent.com/6654265/155041343-bfdb51f7-863e-4c8c-9990-d3d749356479.png)

**Reply by zachleat** [reply]
I'm a bit confused. When you're working in the repo you need the devDeps but they don't install when folks use Eleventy in their projects

**Reply by genemars** [reply]
right, I was just trying to guess what other dependencies could be dropped along with this #2242 but it's ok :)

**Reply by genemars** [reply]
When importing github projects from StackBlitz or similar services, or when running from a local container, reducing the size can speed up the boot-up time process.
```
eleventy$ npx howfat @vue/server-renderer vue @11ty/eleventy-plugin-vue
Dependencies: 234
Size: 64.43mb
Files: 3356
```
